tests/sentry/ratelimits/test_redis.py

```diff
@@ -13,3 +15,34 @@ def test_project_key(self):
     def test_simple_key(self):
         assert not self.backend.is_limited("foo", 1)
         assert self.backend.is_limited("foo", 1)
+
```

Comment:
If I'm reading these tests correctly checking if the backend `is_limited` is what increments the counter? Is this the expected behavior? If so, is there a way we can test that endpoints that have the RateLimiter do in fact run the `is_limited` check on every API call?

Reply:
`is_limited` increments the rate limit counter. 
With the middleware, every endpoint will run is_limited as a part the rate limit check. We'd basically be testing that middlewares do what they advertise :P